pkgs/development/libraries/boxfort/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, coreutils, fetchFromGitHub, cmake, meson, ninja, pkg-config, python3, git }:
+
+stdenv.mkDerivation rec {
+  pname = "boxfort";
+  version = "38fe630";
+
+  src = fetchFromGitHub
+    { owner = "Snaipe";
+      repo  = "BoxFort";
+      rev   = "38fe63046fbabcae34ebc2ee9867d990ac28c4c5";
+      sha256 = "03zy982d1frpk9fr4hbp1ql1ah06s0v6dy7a56d1zl3jjjljhrhn";
+    };
+
+  buildInputs =
+    [ meson
+      ninja
+      coreutils
+      cmake
+      pkg-config
+      python3
+      git
+    ];
+
+  preConfigure = ''
+    mkdir -p /usr/bin
+    ln -s ${coreutils}/bin/env /usr/bin/env
```

Comment:
:thinking:

Why?

Reply:
There's a shebanged script in the `boxfort` build process and I couldn't make it work in a non-NixOS environment for some reason.

Reply:
the right thing would be `patchShebang [<script>]` instead

Reply:
not to mention, this causes an error in sandbox environments
```
builder for '/nix/store/5adjl3031aycx550g63qvh2ai1xvi2mq-boxfort-unstable-2019-09-19.drv' failed with exit code 1; last 6 log lines:
  unpacking sources
  unpacking source archive /nix/store/g8y8nw5v8d20s69kmb61ldqs6y3czcpr-source
  source root is source
  patching sources
  configuring
  mkdir: cannot create directory '/usr': Permission denied
```

Reply:
you should use `patchShebang [<script>]`